scripts/pkgdep.sh

```diff
@@ -34,11 +36,12 @@ INSTALL_CRYPTO=false
 INSTALL_DEV_TOOLS=false
 INSTALL_PMEM=false
 INSTALL_FUSE=false
+INSTALL_RBD=false
 INSTALL_RDMA=false
 INSTALL_DOCS=false
 INSTALL_LIBURING=false
 
-while getopts 'abdfhipru-:' optchar; do
+while getopts 'abdfhipruDR-:' optchar; do
```

Comment:
```suggestion
while getopts 'abdfhipruR-:' optchar; do
```
That's also in the other PR